### PR TITLE
README: General improvements.

### PR DESCRIPTION
**Pull request by Flonky**
Changes:
- images are now at the bottom
- better and more detailed install instructions
- index added

### CODE REVIEW

**Comment by adityaruplaha** [comment]
Oh I just merged my PR which conflicts with yours. Sorry. Please rebase.

File: README.md [hunk 1]
```diff
@@ -6,23 +6,28 @@ Heroic is built with Web Technologies like: TypeScript, React, NodeJS and Electr
...
-![image](https://user-images.githubusercontent.com/26871415/108600451-8eebb500-7397-11eb-966a-70849a589902.png)
-![image](https://user-images.githubusercontent.com/26871415/108600462-a460df00-7397-11eb-8a42-cde5b9b2744c.png)
-![image](https://user-images.githubusercontent.com/26871415/108600516-e2f69980-7397-11eb-8b96-513729859b86.png)
+- [How to use it](#how-to-use-it)
+- [Feature available right now](#feature-available-right-now)
+- [Planned features](#planned-features)
+- [Language Support](#language-support)
+- [Installation](#installation)
+  - [Debian](#debian)
+  - [Arch (Aur)](#arch-aur)
```
Comment (adityaruplaha) [inline README.md:19]: Please capitalise AUR.

File: README.md [hunk 2]
```diff
@@ -61,21 +66,77 @@ Heroic is built with Web Technologies like: TypeScript, React, NodeJS and Electr
...
+
+cd HeroicGamesLauncher
+
+yarn
+
+yarn start
+
+yarn electron
+
+yarn dist {package to create}
```
Comment (adityaruplaha) [inline README.md:132]: This is pretty ambiguous, please change the headings to "Build binaries locally.".
Also stuff for macOS will be added soon.

**Comment by adityaruplaha** [comment]
Please resolve conflicts and we can merge.

**Review by adityaruplaha — APPROVED** [review]

**Review by flavioislima — APPROVED** [review]

**Pull request merged**